frontend/blog/2023-07-03-new-frontend.md

```diff
@@ -0,0 +1,21 @@
+---
+slug: new-frontend
+title: We launched a new frontend
```

Comment:
We joked about the silly first blog post I put in the proof-of-concept site I made to evaluate this tech.

I actually think a blog on shields.io is quite a good place to put things like:

- We're making some kind of unavoidable breaking change to shields.io (e.g: simple-icons major update with removed icons)
- We're deprecating thing X for self hosting users (e.g: removal of redis token pool)
- We just put out a new package version - here's the release notes

i.e: the kind of thing might usually put in pinned issues. I don't think that means we stop using pinned issues, but having somewhere on shields.io to put those "announcements" is useful IMO.

I do have other ideas for some things we might post (if we have the time/inclination). Some possible topics or post formats might include:

- Behind the scenes: Donations + Spending. What we get, how we spend it. How you can help.
- Here's a cool thing someone did with shields that we like (e.g: https://upptime.js.org/docs/badges/ )
- We have an awesome list: https://github.com/badges/awesome-badges submit your stuff to it
- Say thanks to one of our high value sponsors

..but I see giving us a place for those "announcement" posts as being the primary thing this is useful for.

I've put this here as a first post. Tbh, it is mainly just here to "seed" the blog with something. Gotta start somewhere. If we don't like this, maybe we could just put https://github.com/badges/shields/issues/9214 here as a back-dated post?